src/org/jitsi/impl/neomedia/rtp/TransportCCEngine.java

```diff
@@ -361,5 +386,70 @@ private RTCPTransformer()
         }
     }
 
+    /**
+     * {@Link BITRATE_UPDATE_INTERVAL} is an interval used to
+     */
+    private static final int BITRATE_UPDATE_INTERVAL = 5;
```

Comment:
I am removing this value for now. I initially thought the calculation of the receivers tcc remote bit rate at the bridge will be done periodically at the Sender.